### ISSUE TITLE
Add screenshots to introduction

### ISSUE THREAD

**Issue by herbertblb** [erictapen/mapping-migration#66]
@seragarez, would you prepare screenshots of the final states?
I then can try to include them (I do not know how it works though, but I will try to find out)

**Comment by seragarez** [comment]
Yes, will do that! But I am still waiting in case further changes are coming.. until when do you think you need them at the latest?

**Comment by herbertblb** [comment]
around 5 maybe

**Comment by herbertblb** [comment]
also, when I open the app, the time slider is blue, is that how it´s supposed to be  @erictapen ? Because in the screenshot on Miro it´s grey

**Comment by erictapen** [comment]
@herbertblb The screenshot of the timeslider is a non-decision yet unfortunately... Never really looked into how it could be styled.
So if its greyscale on some images that should be solely due to browser-dependent behaviour.
